.github/workflows/testing-all-oses.yml

```diff
@@ -29,6 +28,7 @@ jobs:
           sed -e "s/menuinst.*//" |
           sed -e "s/.*://" > requirements.tmp.txt
         cat requirements.d/development.txt >> requirements.tmp.txt
+        echo "pytest-randomly" >> requirements.tmp.txt
```

Comment:
We maybe should add this to the https://github.com/Open-MSS/MSS/blob/develop/requirements.d/development.txt

Reply:
Yes, once it is installed it will randomize the test order by default, unless you pass `--randomly-dont-reorganize`. That is why I decided to only add it in this workflow, until the test suite fully works with it.